manage/client.go

```diff
@@ -36,6 +36,8 @@ import (
 	"github.com/snapcore/snapd/asserts"
 )
 
+var deviceKey = "./keystore/TestDeviceKey.asc"
```

Comment:
var or const?

Reply:
It's var as it is overwritten in the test. We used the same trick for IndexTemplate in handlers_app.go.